Add limitations on Amazon RDS for MSSQL Server when enabling MSDTC (#6796)
Particular/docs.particular.net: nservicebus/operations/transactions-message-processing.md

Refer to the [Message Handling Pipeline](/nservicebus/pipeline/) article to learn more about message processing.

Refer to the [Recoverability](/nservicebus/recoverability/) and the [ServicePulse: Failed Message Monitoring](/servicepulse/intro-failed-messages.md) articles to learn more about error handling, automatic and manual retries, as well as processing failures monitoring.

## Limitations

> If you are [enabling MSDTC on Amazon RDS for SQL Server](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.SQLServer.Options.MSDTC.html), there are a set of [limitations](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.SQLServer.Options.MSDTC.html#Appendix.SQLServer.Options.MSDTC.Limitations) you should take into account.

Reviewer: I prefer to avoid the `you` when writing documentation

```suggestion
When [enabling MSDTC on Amazon RDS for SQL Server](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.SQLServer.Options.MSDTC.html), a set of [limitations](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.SQLServer.Options.MSDTC.html#Appendix.SQLServer.Options.MSDTC.Limitations) must be taken into account.
```
